src/js/locale/pl.yaml

```diff
@@ -0,0 +1,532 @@
+name: Polski
+
+##
+# Common and reusable phrases
+##
+errors:
+  uri_not_found: 'Nie rozpoznano URI %{uri}'
+  could_not_load: 'Nie można załadować %{name}'
+  could_not_load_library: 'Nie można załadować %{name} od %{provider}'
+  no_results: Nic nie znaleziono
+  authorization_required: 'Musisz najpierw zautoryzować %{provider}'
+  enable_first: 'Odblokuj najpierw %{provider}'
+  no_service_worker: Nie wspierany service worker

```

Comment:
Probably should be `Niewspierany`.
Also I suppose there is no translation for `service worker`